README.md

```diff
@@ -75,22 +60,42 @@ In Go land:
 
 ### Node.js
 
-Install `interface-ipfs-core` as one of the dependencies of your project and as a test file. Then, using `mocha` (for Node.js) or a test runner with compatible API, do:
+Install `interface-ipfs-core` as one of the dependencies of your project. Install the following test file:
 
-```
-var test = require('interface-ipfs-core')
+```js
+/* eslint-env mocha */
+'use strict'
+
+const test = require('interface-ipfs-core')
+const IPFS = require('ipfs')
```

Comment:
it should be made clear that this does should be your own ipfs instance and not the package `ipfs`.